lxqt-admin-user/CMakeLists.txt

```diff
@@ -4,7 +4,12 @@ project(lxqt-admin-user)
 include_directories (
     ${CMAKE_CURRENT_BINARY_DIR}
 )
-
+if(${CMAKE_SYSTEM_NAME} MATCHES  "FreeBSD")
+    set(OS_FREEBSD 1)
+else()
+    set(OS_FREEBSD 0)
+endif()
+configure_file("system_detect.h.in" "${PROJECT_SOURCE_DIR}/system_detect.h")
```

Comment:
Hm... on my system (Qt5.7) I see this in the moc_mainwindow.cpp:
```c++
static const qt_meta_stringdata_MainWindow_t qt_meta_stringdata_MainWindow = {
    { 
QT_MOC_LITERAL(0, 0, 10), // "MainWindow"
QT_MOC_LITERAL(1, 11, 5), // "onAdd"
QT_MOC_LITERAL(2, 17, 0), // ""
QT_MOC_LITERAL(3, 18, 8), // "onDelete"
QT_MOC_LITERAL(4, 27, 16), // "onEditProperties"
QT_MOC_LITERAL(5, 44, 14), // "onChangePasswd"
QT_MOC_LITERAL(6, 59, 6), // "reload"
QT_MOC_LITERAL(7, 66, 14), // "onRowActivated"
QT_MOC_LITERAL(8, 81, 5) // "index"

    },
    "MainWindow\0onAdd\0\0onDelete\0onEditProperties\0"
    "onChangePasswd\0reload\0onRowActivated\0"
    "index"
};
```
=> no `onTabChange`